lego/apps/meetings/tasks.py

```diff
@@ -0,0 +1,41 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from structlog import get_logger
+
+from lego import celery_app
+from lego.apps.meetings import constants
+from lego.apps.meetings.models import Meeting, MeetingInvitation
+from lego.apps.meetings.notifications import MeetingInvitationReminderNotification
+from lego.utils.tasks import AbakusTask
+
+log = get_logger()
+
+
+@celery_app.task(serializer="json", bind=True, base=AbakusTask)
+def notify_user_of_unanswered_meeting_invitation(self, logger_context=None):
+    self.setup_logger(logger_context)
+
+    meetings: list[Meeting] = Meeting.objects.filter(
+        start_time__gt=timezone.now(),
+        # Only send reminder 2 days after creation
+        created_at__lt=timezone.now() - timedelta(days=2),
+    )
```

Comment:
Could it be an idea to limit this to a timeframe as well? So that you will only be notified if the meeting is in the next x days?

For context I just thought about inviting to a series of meetings with regular frequency - and that it could be nice to just create all the meetings in the beginning of the semester and for people to answer it 1-2 weeks in advance when they know what they'll be up to

Reply:
Yep, that can also be done😄 How long before do you think is suitable? Two days?

Reply:
Whopsies, forgot to reply to this one.. How about notifying the recipients if they haven't answered, starting 7 days in advance, and then notifying them every other day until the day before? so 7, 5, 3, 1 days before the date of the meeting.

Reply:
Sure! I'll see what I can do!

Reply:
It now only sends reminder when it's less than 7 days and either 7, 5, 3 or 1 day till start.